packages/mobile/src/walletConnect/walletConnect.ts

```diff
@@ -36,7 +41,7 @@ export function* handleWalletConnectDeepLink(deepLink: string) {
   // handler is called, so it's important we don't display the loading screen on top
   const hasPendingState: boolean = yield select(selectHasPendingState)
   if (!hasPendingState) {
-    navigate(Screens.WalletConnectLoading, { origin: WalletConnectPairingOrigin.Deeplink })
+    handleWalletConnectLoadingWithTimeout({ origin: WalletConnectPairingOrigin.Deeplink })
```

Comment:
We should use `yield call(...)`  since this is a side effect.
Otherwise it's more difficult to unit test.

Reply:
ah yes, i have updated :)